csunplugged/topics/content/en/image-representation/unit-plan/lessons/colour-by-numbers.md

```diff
@@ -0,0 +1,157 @@
+# Colour by numbers
+
+
+## Key questions
+
+- How do you think computers display images on a screen?
+- How do computers store images when they store all information as digits?
+
+
+## Lesson starter
+
+
+Children will need:
```

Comment:
This information is usually only written in the sidebar, not in the main content block.

Reply:
Whoops missed that! I'll move those, thanks.